pom.xml

```diff
@@ -321,7 +321,6 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>9.2.0</version>
```

Comment:
Latest version 10.0.2 will be pulled in from parent https://github.com/okta/okta-java-parent/blob/master/pom.xml#L153